tf-repo-mgmt/repository_sync/github.rulesets.tf

```
@@ -35,3 +35,43 @@ resource "github_repository_ruleset" "main" {
     }
   }
 }
+
+resource "github_repository_ruleset" "tag_deny_version_without_v" {
+  name        = "No version tags without 'v' prefix"
+  repository  = data.github_repository.this.name
+  target      = "tag"
+  enforcement = "active"
+
+  rules {
+    creation = true
+    update   = true
+  }
+
+  conditions {
+    ref_name {
+      include = ["refs/tags/[0123456789]*"]

```

```suggestion
      include = ["*"]
      exclude = ["refs/tags/v[0-9]*.[0-9]*.[0-9]*
```

Could something like this be a bit more restrictive?